arangod/Aql/AqlFunctionFeature.cpp

```diff
@@ -67,9 +67,10 @@ void AqlFunctionFeature::prepare() {
 void AqlFunctionFeature::add(Function const& func) {
   TRI_ASSERT(func.name == basics::StringUtils::toupper(func.name));
   _functionNames.doUnderLock([&](auto& functions) {
-    TRI_ASSERT(!functions.contains(func.name));
-    // add function to the map
-    functions.try_emplace(func.name, func);
+    if (!functions.contains(func.name)) {
+      // add function to the map
+      functions.try_emplace(func.name, func);
+    }
```

Comment:
`try_emplace` will do nothing, if the [key already exists](https://en.cppreference.com/w/cpp/container/map/try_emplace). 
> If a key equivalent to k already exists in the container, does nothing.

Reply:
Another thing, why do we have to guard against this? Shouldn't a function only ever be added once? 🤔